packages/bbui/src/Form/Multiselect.svelte

```diff
@@ -27,7 +27,7 @@
     {error}
     {disabled}
     {readonly}
-    {value}
+    value={Array.isArray(value) ? value : [value]}
```

Comment:
I see this is already merged so we can address this in the other branch -  but this expression will coerce nullish values into non-nullish values which can have unintended consequencies. `null` becomes `[null]` which is now not nullish, and the same for `undefined`.

It would be a better idea to use the existing `onOperatorChange` callback to ensure the value is parseable as an array one this operator is selected, and clear the value if it isn't. That's the purpose of those callbacks when changing fields or operators - to ensure the other properties of the expression are still sensible.

Reply:
Makes sense!

Reply:
I've made that change in the other PR/branch!